### PR TITLE
Implementing new params for vcd cluster create command

### PR DESCRIPTION
**Pull request by rocknes**
Testing done :
CLI command :
(vcd-cli) C:\code\vcd-cli>vcd cluster create -n net1 -N 2 -c 2 -m 1025 --ssh-key ssh-key.txt "aritra-test-cluster"

cse.log :
DEBUG    2017-11-29 08:14:08,615 container_service_extension.processor    process_request                      55  : request body: {"name": "aritra-test-cluster", "node_count": 2, "vdc": "vdc1", "cpu-count": "2", "memory": "1025", "network": "net1", "storage_profile": null, "ssh-key": "sample SSH key"}

Note : the command failed with an error stating 'not enough IP addresses' on the extension. But I believe it's not related to my changes.

### CODE REVIEW

File: vcd_cli/cluster.py [hunk 1]
```diff
@@ -88,21 +88,43 @@ def list(ctx):
...
+@click.option('-s',
               '--storage-profile',
+              'storage_profile',
               required=False,
               default=None,
               metavar='<storage-profile>',
               help='Name of the storage profile for the nodes')
-def create(ctx, name, node_count, network_name, storage_profile):
+@click.option('-k',
+              '--ssh_key',
```
Comment (pacogomez) [inline vcd_cli/cluster.py]: `--ssh-key` is preferred

File: vcd_cli/cluster.py [hunk 2]
```diff
@@ -88,21 +88,43 @@ def list(ctx):
...
+              'ssh_key',
+              required=False,
+              default=None,
+              metavar='<ssh_key>',
```
Comment (pacogomez) [inline vcd_cli/cluster.py]: `<ssh-key>` is preferred

File: vcd_cli/cluster.py [hunk 3]
```diff
@@ -88,21 +88,43 @@ def list(ctx):
...
+              help='SSH key to talk to the guest os on the vm')
```
Comment (pacogomez) [inline vcd_cli/cluster.py]: The `ssh-key` should be the name of a file where the public ssh key is stored. the command should read the content of the file and pass it to the `create` method. click file option should be used.

File: vcd_cli/cluster.py (hunk 1 above)
Comment (rocknes) [reply vcd_cli/cluster.py]: updated

File: vcd_cli/cluster.py (hunk 2 above)
Comment (rocknes) [reply vcd_cli/cluster.py]: updated

File: vcd_cli/cluster.py (hunk 3 above)
Comment (rocknes) [reply vcd_cli/cluster.py]: Addressed.

**Pull request merged**